### ISSUE TITLE
Northstar Menu Tabs Not Announcing Selection

### ISSUE THREAD

**Issue by wsaballos** [microsoft/fluentui#21164]
### Environment Information

- **Package version(s)**: 0.58.0
- **Browser and OS versions**: Chrome MacOS

### Describe the issue:

Narrator and VoiceOver are not announcing "selected" after a selection is made on a Tab within the Menu component. 

### Please provide a reproduction of the issue in a codepen:

[Sandbox](https://codesandbox.io/s/fluent-ui-example-forked-h5497?file=/example.js)

#### Actual behavior:
While using narrator, select a tab and observe that there is **no** announcement that a selection has been made.

#### Expected behavior:
Narrator should announce that a selection has been made after selecting a tab by announcing 'Selected'.

**Comment by jurokapsiar** [comment]
Unable to repro. After pressing Enter, Voiceover announces that the focus is on a selected tab:
![image](https://user-images.githubusercontent.com/13742664/149981579-a763acbd-61d4-4af6-a1de-bb55bb28731c.png)
